esp/esp/program/forms.py

```diff
@@ -62,23 +62,46 @@ class ProgramCreationForm(BetterModelForm):
     student_reg_end   = forms.DateTimeField(widget = DateTimeWidget())
     base_cost         = forms.IntegerField(label = 'Cost of Program Admission $', min_value = 0 )
     sibling_discount  = forms.DecimalField(max_digits=9, decimal_places=2, required=False, initial=None,
-                                                help_text="The amount of the sibling discount. Leave blank if you don't use sibling discounts.")
+                                           help_text="The amount of the sibling discount. Leave blank if you don't use sibling discounts.")
     program_type      = forms.CharField(label = "Program Type", help_text='e.g. Splash or Cascade')
-    program_modules   = forms.MultipleChoiceField(
-                          choices=[],
-                          label='Program Modules',
-                          widget=forms.SelectMultiple(attrs={'class': 'input-xxlarge'}),
-                          help_text=Program.program_modules.field.help_text)
+    program_module_questions   = forms.MultipleChoiceField(choices=[],
+                                                           label='Program Modules',
+                                                           widget=forms.CheckboxSelectMultiple(attrs={'class': 'input-xxlarge'}),
+                                                           help_text=Program.program_modules.field.help_text,
+                                                           required=False)
+
+    program_modules   = forms.MultipleChoiceField(choices=[], widget = forms.SelectMultiple(attrs={'class': 'hidden-field'}), required=False)
 
     def __init__(self, *args, **kwargs):
         """ Used to update ChoiceFields with the current modules. """
+        # These modules are the "choosable" ones that admins will usually want to choose to select or exclude (i.e. not automatically include or exclude)
+        self.program_module_question_ids = OrderedDict({'Will you charge for the program?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Accounting', 'Financial Aid Application', 'Easily Approve Financial Aid Requests'])],
+                                                        'Will you have extra costs (shirts or lunch)?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Optional Fees', 'Accounting', 'Financial Aid Application', 'Easily Approve Financial Aid Requests'])],
+                                                        'If you will charge for admission or other costs, will you accept payment by credit card?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Credit Card Payment Module (Stripe)', 'Credit Card View Module'])],
+                                                        'Do you want a pre-program quiz for teachers?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Logistics Quiz', 'Teacher Custom Form'])],
```

Comment:
Do you need the "Teacher Custom Form" module enabled in order to have the "Teacher Logistics Quiz" module work?

Reply:
Nope. I was thinking that the quiz is a custom form, but making one doesn't require the module. Fixed in commit cdeabbf